packages/tasks/src/local-apps.ts

```diff
@@ -262,6 +262,43 @@ const snippetTgi = (model: ModelData): LocalAppSnippet[] => {
 	];
 };
 
+const snippetMlxLm = (model: ModelData): LocalAppSnippet[] => {
+	const openaiCurl = [
+		"# Calling the OpenAI-compatible server with curl",
+		`curl -X POST "http://localhost:8000/v1/chat/completions" \\`,
+		`   -H "Content-Type: application/json" \\`,
+		`   --data '{`,
+		`     "model": "${model.id}",`,
+		`     "messages": [`,
+		`       {"role": "user", "content": "Hello"}`,
+		`     ]`,
+		`   }'`,
+	];
+
+	return [
+		{
+			title: "Generate or start a chat session",
+			setup: ["# Install MLX LM", "pip install mlx-lm"].join("\n"),
```

Comment:
or use uvx and drop the pip install?

or even better, `uv tool install mlx-lm`